frontend/src/features/org/org-status-banner.ts

```diff
@@ -93,11 +90,12 @@ export class OrgStatusBanner extends BtrixElement {
         content: () => {
           return {
             title:
-              hoursDiff < 24
-                ? msg("Your org will be deleted within one day")
-                : daysDiff === 1
-                  ? msg("Your org will be deleted in one day.")
-                  : msg(str`Your org will be deleted in ${daysDiff} days`),
+              daysDiff > 1
+                ? msg(
+                    str`Your org will be deleted in
+              ${daysDiff} days`,
+                  )
+                : `Your org will be deleted within one day`,
```

Comment:
It looks like the merge conflict was resolved incorrectly, removing the changes from https://github.com/webrecorder/browsertrix/pull/2193.